src/otx/recipe/anomaly_classification/draem.yaml

```diff
@@ -31,11 +32,10 @@ overrides:
         batch_size: 8
         num_workers: 4
         transforms:
-          - class_path: otx.core.data.transform_libs.torchvision.ResizetoLongestEdge
```

Comment:
what is the difference between the native torchvision transform and `otx.core.data.transform_libs.torchvision`. Ideally transforms can be imported directly from `torchvision.transforms.v2.Resize`? @harimkang any thoughts?

@ashwinvaidya17, this way of transform configuration should ideally be in anomalib too

Reply:
Um, looking at it, this looks like a transform for the visual_prompting model, @ashwinvaidya17 is there a reason to use this? we can use `torchvision.transforms.v2.Resize`